ai-ml/maxdiffusion/build/server/cloudbuild.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START gke_ai_ml_maxdiffusion_build_server_cloudbuild]
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'us-east1-docker.pkg.dev/$PROJECT_ID/gke-llm/max-diffusion:latest', '.' ]
+images:
+- '$LOCATION-docker.pkg.dev/$PROJECT_ID/gke-llm/max-diffusion:latest'
```

Comment:
This file as is doesn't work for me. I tried using $_REGION-docker.pkg.dev/$PROJECT_ID/gke-llm/max-diffusion:latest and passing in the $REGION value exported by the user earlier in the guide like gcloud builds submit . --substitutions _REGION=$REGION